src/guidestyle.css

```diff
@@ -2,3 +2,83 @@ DIV.version {text-align: right ; font-style: italic }
 TD.aligned  {vertical-align: top ; padding-right: 1em }
 H3 { font-size: 10pt; font-weight: bold; font-style: italic;
      padding: 0pt; margin: 1.5ex 0pt 0pt 0pt; }
+
+
+/* Use a section symbol as list item for style guide items */
+div.conventions > ul {
+  list-style: none;
+  margin-left: 0;
+  padding-left: 0;
+}
+
+div.conventions > ul > li {
+    padding-left: 1em;
+    text-indent: -1em;
+    padding-bottom: 1em;
+}
+
+div.conventions > ul > li:before {
+    content: '\a7';
+    padding-right: 5px;
+    color: #5382a1;
+}
+
+div.conventions > * > li > p:first-child {
+    display: inline;
+}
+
+/* Reset styles that would otherwise be inherited from div.conventions > li */
+div.conventions > ul > li > * {
+    text-indent: 0em;
+}
+
+.dos {
+    color: #008000;
+}
+
+.donts {
+    color: #993300;
+}
+
+/* Spacing between consecutive code examples */
+pre ~ pre {
+    margin-top: 1em;
+}
+
+/* Code blocks */
+pre > code {
```

Comment:
Good point. I grep'ed for `<code` and found out that this styling messed up code blocks in hgHelp and tClone. Updated those files.

I also noted that code blocks are formatted using blockquotes here and there. Perhaps this is because there hasn't been any proper styling for code blocks before? I think they should be turned into `~~~`, but I'll address that in a separate change.